chromadb/test/property/test_collections_with_database_tenant.py

```diff
@@ -36,7 +38,15 @@ class TenantDatabaseCollectionStateMachine(CollectionStateMachine):
     def __init__(self, client: Client):
         super().__init__(client)
         self.api = client
-        self.admin_client = AdminClient.from_system(client._system)
+
+        if (
```

Comment:
Can we rely on client factories or similar rather than making this text fixture aware?

Reply:
yep, a little awkward but fixed in e94fb90f